src/main/webapp/scripts/tests/controller-auth-test.js

```diff
@@ -0,0 +1,93 @@
+import * as view from '/scripts/view.js';
+import * as controller from '/scripts/controller-auth.js';
+
+// Test (controller-auth.js and) view.js
+QUnit.module('Authorization controller', {
+  beforeEach: () => {
+    gapi = new CustomGapi();
+  },
+  afterEach: () => {
+    view.updateSigninStatus(false, null, null, null);
+    document.getElementById('result-text').innerText = '';
+    sinon.restore();
+  }
+});
+
+QUnit.test('When init fails, displays the authorization button and the error message', async function (assert) {
+  const errorMsg = 'error message';
```

Comment:
Nit: Seems like this message is irrelevant to the test. Below, you could just do `new Error()`

Reply:
This error message actually shows up in DOM and it changes when deployed (video not found, id cannot be empty etc)

Reply:
oow wait I messed up you are right haha